projects/marketing-site/public/lang/common.json

```diff
@@ -0,0 +1,242 @@
+            "proposal": "Đề xuất",
+            "team": "Đội ngũ",
+            "feature_title": "Tất cả tính năng trong một nền tảng",
+            "feature_desc": "Chúng tôi đã thẩm định và xây dụng một sàn giao dịch NFT vì cộng đồng, dưới đây là các tính năng của sản phẩm."
+        }
+    ],
+    "vision_feature": [
+        {
+            "locale": "en-US",
+            "name": "Hyperspeed",
+            "description":
+                "By using the latest tech and positioning our servers around the world, Hyperdive will load fast and run smooth."
+        },
+        {
+            "locale": "en-US",
+            "name": "Creator Focused",
+            "description":
+                "Customizable NFT Creator pages that bring a storefront to life and combine collections and off-chain activity."
+        },
+        {
+            "locale": "en-US",
+            "name": "Bot Protection",
+            "description":
+                "We want your marketplace to grow organically and help ensure we are always open for business."
+        },
+        {
+            "locale": "en-US",
+            "name": "Expert Team",
+            "description":
+                "We come from well known projects on L2's and have official backgrounds in engineering and finance."
+        },
+        {
+            "locale": "en-US",
+            "name": "Open Source",
+            "description":
+                "Transparency is so key to crypto and you deserve to know when you buy with us, what you see is what you get."
+        },
+        {
+            "locale": "en-US",
+            "name": "Community Maintained",
+            "description":
+                "Want to add a feature or integrate an api to help improve the Marketplace? Hack around and find out, fren!"
+        },
+        {
+            "locale": "en-US",
+            "name": "Multilingual",
+            "description":
+                "You should be able to experience the Marketplace as you would in your own town."
+        },
+        {
+            "locale": "en-US",
+            "name": "Social Media",
+            "description":
+                "With a focus on decentralized social media solutions, we intend to increase retention on Creator pages."
+        },
+        {
+            "locale": "vi-VN",
+            "name": "Siêu tốc",
+            "description":
+                "Bằng việc sử dụng công nghệ mới nhất và có hệ thống máy chủ trên toàn cầu. Trải nghiệm trên Hyperdrive sẽ nhanh và mượt."
+        },
+        {
+            "locale": "vi-VN",
+            "name": "Người sáng tạo nội dung",
+            "description":
+                "Sản phẩm cho phép người sáng tạo nội dung tuỳ chỉnh NFT và đăng bộ sưu tập của họ lên cửa hàng"
+        },
+        {
+            "locale": "vi-VN",
+            "name": "Bot Protection",
```

Comment:
is there no real translation for "Bot Protection" in Vietnamese?

Reply:
Yes, i think so